test/test_linearizer.py

```diff
@@ -106,6 +107,21 @@ def test_sum_collapse(self):
     lin = Linearizer(sched[0].ast)
     assert not any(u.uop == UOps.LOOP for u in lin.linearize().uops), "found loop in sum collapse"
 
+  reduce_ops = (Tensor.max, Tensor.min, Tensor.sum)
+  float_dtypes = [v for v in DTYPES_DICT.values() if dtypes.is_float(v)]
```

Comment:
consider putting these in `test/helpers.py`. also i removed `dtypes._arg_int32` that might simplify some logic

Reply:
we need it because bool exists